test-packages/nightly-tests/generator/openapi-negative-case.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { resolve } from 'path';
+import execa from 'execa';
+import {
+  testDir,
+  testOutputRootDir,
+  testResourcesDir
+} from '../../../test-resources/generator';
+
+describe('openapi negative tests', () => {
+  // TODO Put test back in once the spec validation is put back in.
+  xit('should fail on generation for faulty spec file', async () => {
+    const output = resolve(
+      testOutputRootDir,
+      'openapi-negative',
+      'faulty-specification'
+    );
+    await expect(
+      execa(
+        'npx',
+        [
+          'generate-openapi-client',
+          '-i',
+          resolve(testResourcesDir, 'test-resources', 'faulty-openapi'),
+          '-o',
+          output,
+          '--clearOutputDir'
+        ],
+        { cwd: __dirname }
+      )
+    ).rejects.toThrowError('set failed method');
```

Comment:
Could you please share more details about why this spec is invalid, because the error is not descriptive. The information can be added to the description of the spec then.